systemtests/src/main/java/io/enmasse/systemtest/messagingclients/AbstractClient.java

```diff
@@ -42,20 +46,21 @@
     private List<String> executable;
     private String podName;
     private String podNamespace;
+    private CompletableFuture<Void> clientAttached;
```

Comment:
Naming - it is not the client that is attached, but link, so I think linkAttached would be better.

Reply:
changed